### ISSUE TITLE
Supporting screen reader with multiple links in a paragraph

### ISSUE THREAD

**Issue by timoisalive** [software-mansion-labs/react-native-enriched-markdown#50]
**Is your feature request related to a problem? Please describe.**
We are obliged to support accessibility features in our app. One of the problems has been screen reader not finding links in rich text.

`react-native-enriched-markdown` seems to support one link in a paragraph by making the whole paragraph actionable and reading out the accessibility hint to double tap. That's great, but another link within the same paragraph removes the hint and still performs the action, only on the first link though.

**Describe the solution you'd like**
No libraries (that we know of) seem to support links. The issue usually is that the screen reader finds the block of text in which the link is, and reads is as whole. It doesn't understand there's a link inside. One solution is to break the block of text into multiple blocks (siblings: text, link, text), but that introduces text layout problems.

Web browsers are capable of handling links as they should be handled; when navigating the text with screen reader, it stops into links and reads them out (and allows performing the actions of course).

**Comment by hryhoriiK97** [comment]
@timoisalive the PR with screen reader support has been merged. We're hoping to release it next week 🙂
